lib/public/AppFramework/Http/Attribute/IgnoreAPI.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+declare(strict_types=1);
+
+/**
+ * @copyright Copyright (c) 2023 Kate Döen <kate.doeen@nextcloud.com>
+ *
+ * @author Kate Döen <kate.doeen@nextcloud.com>
+ *
+ * @license GNU AGPL version 3 or any later version
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as
+ * published by the Free Software Foundation, either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+namespace OCP\AppFramework\Http\Attribute;
+
+use Attribute;
+
+/**
+ * Attribute for controller methods that should be ignored when generating API documentation
+ *
+ * @since 28.0.0
+ */
+#[Attribute]
```

Comment:
```suggestion
#[Attribute(Attribute::TARGET_METHOD)]
```

Or does it apply to full classes as well?

Reply:
Right now no, but now that you mention it it would make a lot of sense. Will implement that, so the attribute will also work for controllers :+1:

Reply:
Then:
```suggestion
#[Attribute(Attribute::TARGET_METHOD | Attribute::TARGET_CLASS)]
```